alpha-blog/app/views/articles/_messages.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <% flash.each do |name, msg| %>
-    <div class="alert alert-success alert-dismissible fade show" role="alert">
+    <div class="alert alert-<%= name == "notice" ? "success" : "danger" %> alert-dismissible fade show" role="alert">
```

Comment:
Why not just:
```
<div class="alert alert-<%= name %> alert-dismissible fade show"
```
?

Reply:
the name passed in is either `notice` or `alert` but the bootstrap class name is `alert-success` for green colour and `alert-danger` for red. so alert-<%= name %> will make the class name equals to either `alert-notice` or `alert-alert` which wont implement the bootstrap class name

Reply:
Huh?
You can't set the value of `name` ?

Reply:
nope. its using rails [helper method](https://www.rubyguides.com/2019/11/rails-flash-messages/) `flash[:notice]` or `flash.now[:alert]` that is where the `notice` and `alert` comes from

Reply:
I see.
Wdyt of creating a helper method to handle this rather than doing if else here? 
Whta if you need the same thing somewhere else?

Reply:
🤔🤔okok let me try to do that tommorow👍